package-lock.json

```diff
@@ -7652,25 +7652,6 @@
         "lodash.uniq": "^4.5.0"
       }
     },
-    "node_modules/caniuse-lite": {
```

Comment:
hey @sandramsc , can you please revert this file??

Reply:
@iArchitSharma just so I understand the request, you mean to remove it yes?